configure.ac

```diff
@@ -352,6 +352,33 @@ else
     AC_MSG_RESULT([yes])
 fi
 
+##########################
+# KVM vmapoll workaround
```

Comment:
IMO it should be runtime decision and not compile time:

* detect kvm at runtime and set func ptr to appropriate memcpy
* code use fptr 

it will be nightmare manage binary for kvm or bare metal otherwise.